### PR TITLE
Conan Extension for Visual Studio

### PR DESCRIPTION
**Pull request by jgsogo**
A brief introduction to the plugin, just a kick-off. I think it could be enough if we handle early feedback and write another blogpost talking about CLion one too (_"Conan integrates with your IDE"_). Anyway, if you feel like there should be more wording, just suggest the topic/idea and I'll integrate it into the text.

- [ ] All of you, please, test the plugin in your Visual Studio and report (get last artifact from AppVeyor: https://ci.appveyor.com/project/ConanOrgCI/conan-vs-extension)
- [x] TODO: add a picture from the Extensions Manager inside Visual Studio (need to make the extension public first)
- [ ] Decide publish date

View of the blogpost:

localhost_4000_2019_05_09_Conan-extension-for-Visual-Studio html

### CODE REVIEW

File: _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+to endorse and share with the rest of our users. 
+
+<div align="center">
+    <figure>
+        <img src="{{ site.url }}/assets/post_images/2019-05-09/marketplace-header.png" width="600"/>
+        <figcaption>Conan Extension for Visual Studio in the Microsoft marketplace</figcaption>
+    </figure>
+</div>
+
+This extension provides an smooth integration between Conan and your Visual Studio project. It
```
Comment (danimtb) [inline _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]:
```suggestion
This extension provides smooth integration between Conan and your Visual Studio project. It
```

File: _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+<div align="center">
+    <figure>
+        <img src="{{ site.url }}/assets/post_images/2019-05-09/vs-screen.png" width="600"/>
+        <figcaption>Visual Studio project using Conan Extension</figcaption>
+    </figure>
+</div>
+
+Have a look at the [CLI example](https://github.com/conan-io/conan-vs-extension/tree/master/Conan.VisualStudio.Examples/ExampleCLI)
+available in our repository, and if you have doubts, face any issue or want to provide useful
+feedback, you can reach us in Cpplang Slack (channel #conan), social networks and, of course,
```
Comment (danimtb) [inline _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]: better provide the link to the workspace

File: _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+## Use Conan Extension for Visual Studio
+
+To use this extension you will need a ``conanfile.txt`` or ``conanfile.py`` in your Visual Studio
+solution declaring the dependencies of your project. Conan will download them from the configured
+remotes (or build them if binaries are not available) and generate the `.prop` file that will
+be included in your project with all the paths and flags needed to use and link against that
+required libraries.
+
+Conan will detect the configuration used in your build to select the proper settings for the
+requirements, you can change build type from ``Debug`` to ``Release``, or the toolset and Conan
```
Comment (danimtb) [inline _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]: long sentence and many "you can"

File: _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+<div align="center">
+    <figure>
+        <img src="{{ site.url }}/assets/post_images/2019-05-09/marketplace-header.png" width="600"/>
+        <figcaption>Conan Extension for Visual Studio in the Microsoft marketplace</figcaption>
+    </figure>
+</div>
+
+This extension provides an smooth integration between Conan and your Visual Studio project. It
+will detect a ``conanfile.py`` or ``conanfile.txt`` in the solution, retrieve the requirements
+and generate the `.prop` file to be included in your solution.
```
Comment (danimtb) [inline _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]: We should say something more about the generators "compatible" with the plugin. Is it necessary to declare the generator or the plugin injects it? Can I use other generator? (maybe this last question is out of scope)

Comment (SSE4) [reply _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]: to the record: currently supported are `visual_studio` and `visual_studio_multi`. they are invoked automatically, and props files are injected automatically.

File: _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+ 
+   You can leave the other options with the default values, it isn't the aim of this blog post
+   to enter into those details, but you can read about them and many other features in the
+   [Conan documentation](https://docs.conan.io/en/latest/).
+
+## Use Conan Extension for Visual Studio
+
+To use this extension you will need a ``conanfile.txt`` or ``conanfile.py`` in your Visual Studio
+solution declaring the dependencies of your project. Conan will download them from the configured
+remotes (or build them if binaries are not available) and generate the `.prop` file that will
```
Comment (SSE4) [inline _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]: it's actually `props`, not `prop`

File: _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown (hunk 2 above)
Comment (jgsogo) [reply _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]: Do you mean a link to http://cpplang.slack.com?

File: _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown (hunk 4 above)
Comment (jgsogo) [reply _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]: The idea of this blog post is _plug and play_, and given that you can only use ``visual_studio`` generators I think it is not worth entering into those details.

File: _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown (hunk 2 above)
Comment (danimtb) [reply _posts/2019-05-09-Conan-extension-for-Visual-Studio.markdown]: yes
